### PR TITLE
Updated psij tutorial

### PR DESCRIPTION
**Pull request by hategan**

### CODE REVIEW

File: docs/source/tutorials/psij.ipynb [hunk 1]
```diff
@@ -37,478 +26,614 @@
...
+      "  Stored in directory: /home/jovyan/.cache/pip/wheels/56/e7/49/8561f499b433df173f5ffe04a490876fadc8f51ab62e3a5003\n",
+      "Successfully built psij-python pystache\n",
+      "Installing collected packages: pystache, filelock, psij-python\n",
+      "Successfully installed filelock-3.9.0 psij-python-0.1.0.post2 pystache-0.6.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "pip install git+https://github.com/ExaWorks/psij-python.git"
```
Comment (mtitov) [inline docs/source/tutorials/psij.ipynb:84]: maybe keep original approach to eliminate installation logs?
```
%%capture capt
%pip install ...
```
and with the next cell to show what was installed
```
pip show psij-python
```

**Review by mtitov — APPROVED** [review]
Looks good, runs smoothly!
Have a small comment regarding package installation

File: docs/source/tutorials/psij.ipynb [hunk 2]
```diff
@@ -1,34 +1,23 @@
 {
  "cells": [
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Psij Getting Started Tutorial"
+    "### PSI/J-Python Getting Started Tutorial"
```
Comment (mtitov) [inline docs/source/tutorials/psij.ipynb:7]: and maybe keep this as a title - `# PSI/J-Python...`
(will be in sync with Matteo's PR)

File: docker/tutorials/Dockerfile [hunk 3]
```diff
@@ -30,3 +30,8 @@ RUN conda install -y -c conda-forge radical.entk
 # RP-Parsl Integration usecase nwchem
 
 RUN conda install -c conda-forge nwchem
+
+# Needed for MPI example in PSI/J
+
+RUN conda install -y -c conda-forge gxx_linux-64
+RUN conda install -y -c conda-forge openmpi
```
Comment (mtitov) [inline docker/tutorials/Dockerfile:37]: btw in PR #162 I switched from `conda` to `mamba` (aka advanced `conda`), which is supported by base jupyter docker container, and which helped with parsl-env setup - thus, kinda, either update it here, or later I can update it in that PR (to have it consistent).

and maybe to have this under one `RUN` (== within one layer)?
```
RUN mamba install -y -c conda-forge \
    gxx_linux-64 \
    openmpi
```

File: docs/source/tutorials/psij.ipynb (hunk 1 above)
Comment (hategan) [reply docs/source/tutorials/psij.ipynb]: Indeed that is cleaner. I changed to a variation of that. Thank you.

File: docs/source/tutorials/psij.ipynb (hunk 2 above)
Comment (hategan) [reply docs/source/tutorials/psij.ipynb]: Not sure why the heading got changed there. Fixed. Thanks.

File: docker/tutorials/Dockerfile (hunk 3 above)
Comment (hategan) [reply docker/tutorials/Dockerfile:37]: It looks weird to have two conda install commands and one mamba install command.

But your call. I can make the change here if you think that's better.

Comment (mtitov) [reply docker/tutorials/Dockerfile:37]: yeah, would be good to have it here, thanks!

Comment (hategan) [reply docker/tutorials/Dockerfile:37]: K. I'm doing a sanity test now.

Comment (hategan) [reply docker/tutorials/Dockerfile]: Seems to work.

Comment (mtitov) [reply docker/tutorials/Dockerfile]: Great! ready to go

**Pull request merged**